tests/integrations/raftstore/test_merge.rs

```diff
@@ -213,10 +213,20 @@ fn test_node_merge_prerequisites_check() {
     cluster.must_transfer_leader(left.get_id(), left_on_store1);
     let right_on_store1 = find_peer(&right, 1).unwrap().to_owned();
     cluster.must_transfer_leader(right.get_id(), right_on_store1);
-    cluster.add_send_filter(CloneFilterFactory(RegionPacketFilter::new(
-        left.get_id(),
-        3,
-    )));
+
+    // first MsgAppend will append log, second MsgAppend will set commit index,
+    // So only allowing first MsgAppend to make source peer have uncommitted entries.
+    let append_filter = RegionPacketFilter::new(left.get_id(), 3)
```

Comment:
Now the case only checks committed index.

Reply:
Fixed, PTAL